server/src/pages/shotindex/view.js

```diff
@@ -170,6 +170,9 @@ class Body extends React.Component {
         <Localized id="shotIndexPageConfirmShotDelete">
           <div id="shotIndexPageConfirmShotDelete" hidden></div>
         </Localized>
+        <Localized id="shotPageAlertErrorUpdatingExpirationTime">
```

Comment:
Let's get rid of this

Reply:
Quick clarification, this means we are getting rid of error alert on shotindex. Is that correct?